contracts/HubPool.sol

```diff
@@ -36,6 +37,7 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         uint64 unclaimedPoolRebalanceLeafCount;
         bytes32 poolRebalanceRoot;
         bytes32 destinationDistributionRoot;
+        bytes32 slowRelayFulfilmentRoot;
```

Comment:
nit: I hate to be that grammar guy but fulfillment has two L's in the "fill" part

Reply:
Thank you!

I don't mean to throw @chrismaree under the bus, but I just copy-pasted this from his suggestion haha

Reply:
Will fix.